secured_locker/azure-functions/assign_storage_role.ps1

```diff
@@ -0,0 +1,25 @@
+#Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
If possible, I'd like to also have a bash script equivalent for any powershell scripts (and a powershell script equivalent for any bash scripts). I'm slowly working on adding bash/powershell equivalents where I see one and not the other.

Could you pretty please add a bash script as well? Unless Windows is a requirement for this script.

Reply:
Sure. Will add the bash script later as it requires different command sets which takes some time.

Reply:
If you want to do it as a separate merge request, that's perfectly fine.